website/docs/r/consumption_budget_management_group.html.markdown

```diff
@@ -0,0 +1,180 @@
+---
+subcategory: "Consumption"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_consumption_budget_management_group"
+description: |-
+  Manages a Management Group Consumption Budget.
```

Comment:
Even though the phrasing is consistent with the other consumption budget resources I think this would read better as 
```suggestion
  Manages a Consumption Budget for a Management Group.
```